tests/common.py

```diff
@@ -155,11 +155,12 @@ def prepare_nb_system(
     return params, potential
 
 
-def hilbert_sort(conf, D):
-    hc = HilbertCurve(HILBERT_GRID_DIM, D)
+def hilbert_sort(conf, box):
```

Comment:
@maxentile I changed this to more closely match the behavior in the C++

https://github.com/proteneer/timemachine/blob/master/timemachine/cpp/src/kernels/k_hilbert.cu#L33-L38